Code/Tools/ProjectManager/Source/GemRepo/GemRepoScreen.cpp

```diff
@@ -52,34 +56,70 @@ namespace O3DE::ProjectManager
 
         vLayout->addWidget(m_contentStack);
 
-        Reinit();
+        m_notificationsView = AZStd::make_unique<AzToolsFramework::ToastNotificationsView>(this, AZ_CRC("ReposNotificationsView"));
```

Comment:
This should use `AZ_CRC_CE` for compile time evauluation
```suggestion
        m_notificationsView = AZStd::make_unique<AzToolsFramework::ToastNotificationsView>(this, AZ_CRC_CE("ReposNotificationsView"));
```